Makefile

```diff
@@ -99,8 +97,8 @@ $(BINDIR)/deepcopy-gen: cmd/libs/go2idl/deepcopy-gen
 	$(DOCKER) glide install --strip-vendor
 	touch $@
 
-.scBuildImage: hack/Dockerfile
-	sed "s/GO_VERSION/$(GO_VERSION)/g" < hack/Dockerfile | \
+.scBuildImage: build/build-image/Dockerfile
+	sed "s/GO_VERSION/$(GO_VERSION)/g" < build/build-image/Dockerfile | \
```

Comment:
yea but they have other files in that dir.  A dir for just one file feels really odd to me so I would prefer to wait until we need a "grouping mechanism" before we unnecessarily introduce one.  I would also prefer that we keep all build-related files in one dir, and not split across 'hack' and 'build'.